### ISSUE TITLE
Passing a url does not work

### ISSUE THREAD

**Issue by stuartcusackie** [spatie/statamic-responsive-images#132]
I'm not sure if this is a Blade-only problem but this is not working for me:
`@responsive('images/my-image.png', ['ratio' => '8/5'])`

Neither is this:
`@responsive(asset('images/my-image.png'), ['ratio' => '8/5'])`

Thanks!

**Comment by stuartcusackie** [comment]
oh! No problem then. Statamic's glide does allow us to use any url so I assumed this package allowed the same thing. 

You might consider adding that if possible. I might be missing a reason why it cannot. No problem if not! I can workaround.

**Comment by adevade** [comment]
I'm trying to pass a relative URL to the Antlers tag, but can't get it working.

I'm defining some meta data directly in the yaml of the content file, without hooking it up to the CP.
This is a standard `page`, with some extra one off content for this page only.

I have defined `image: products/my-product.jpg`, and want to pass this through the `responsive` tag.
The image is already uploaded in an asset container and used in other places on the site.

I've tried `{{ responsive:image }}`, `{{ responsive :src="image" }}`, `{{ responsive src="assets/{image}" }}` but nothing seems to be working. It doesn't output anything.

Does it only work when the asset is defined as a field in a blueprint?
